Triphouse-project/index.html

```diff
@@ -27,7 +27,7 @@
                     </li>
```

Comment:
![2021-04-19_22h14_23](https://user-images.githubusercontent.com/19935417/115290800-a1e2f180-a15c-11eb-86b4-718545be95b4.png)
Page is broken on mobile devices.